ldap-client.go

```diff
@@ -41,7 +45,10 @@ func (lc *LDAPClient) Connect() error {
 				return err
 			}
 		} else {
-			l, err = ldap.DialTLS("tcp", address, &tls.Config{InsecureSkipVerify: false})
+			l, err = ldap.DialTLS("tcp", address, &tls.Config{
+				InsecureSkipVerify: lc.InsecureSkipVerify,
```

Comment:
OK I did some more testing and it's the only way to make it work with `useSSL=true`. Thanks a ton for your patience and for the PR!